src/Symfony/Component/Form/Extension/DataCollector/FormDataCollector.php

```diff
@@ -88,12 +88,12 @@ public function reset(): void
 
     public function associateFormWithView(FormInterface $form, FormView $view): void
     {
-        $this->formsByView[spl_object_hash($view)] = spl_object_hash($form);
+        $this->formsByView[spl_object_id($view)] = spl_object_id($form);
     }
 
     public function collectConfiguration(FormInterface $form): void
     {
-        $hash = spl_object_hash($form);
+        $hash = spl_object_id($form);
```

Comment:
Maybe we should rename the variable here and below then? Because it's not a hash anymore

Reply:
Good suggestion